File: src/androidTest/java/pt/joaomneto/titancompanion/TestPOE.java
package pt.joaomneto.titancompanion;


import android.support.test.filters.LargeTest;
import android.support.test.runner.AndroidJUnit4;

import org.junit.runner.RunWith;

import pt.joaomneto.titancompanion.consts.FightingFantasyGamebook;


@LargeTest
@RunWith(AndroidJUnit4.class)
public class TestPOE extends TestST {

    @Override
    protected FightingFantasyGamebook getGamebook() {
        return FightingFantasyGamebook.PORTAL_OF_EVIL;
    }
}

File: src/androidTest/java/pt/joaomneto/titancompanion/TestPOE.kt
package pt.joaomneto.titancompanion

import androidx.test.ext.junit.runners.AndroidJUnit4
import androidx.test.filters.LargeTest
import org.junit.runner.RunWith
import pt.joaomneto.titancompanion.consts.FightingFantasyGamebook

@LargeTest
@RunWith(AndroidJUnit4::class)
class TestPOE : TestST() {

    override val gamebook = FightingFantasyGamebook.PORTAL_OF_EVIL
}
